src/bindings/c/docs/OpenVINO_Runtime_C_API_Developer_Guide.md

```diff
@@ -0,0 +1,84 @@
+# Developer Guide of OpenVINO C API
+
+> **NOTE**: This guide provides a instroduction about OpenVINO C API.
+C API provides simplified interface for OpenVINO functionality that allows to:
+
+- handle the input network models & input/output tensors.
+- load and configure OpenVINO plugins based on device name.
+- perform inference in synchronous and asynchronous modes with arbitrary number of infer requests (the number of infer requests may be limited by target device capabilities).
+
+## Supported OSes
+
+Currently the OpenVINO C API is supported on Ubuntu* 20.04, Microsoft Windows* 10 and CentOS* 7.3 OSes.
```

Comment:
MacOS should also be supported.

Reply:
I don't think that we need to have this guide. Developers can use different OSs